packages/macros/src/index.ts

```diff
@@ -59,6 +59,10 @@ export function failBuild(message: string): void {
   throw new Oops(message);
 }
 
+export function maybeAttrs<T>(predicate: boolean, ...bareAttrs: unknown[]): T {
```

Comment:
Not sure if this function belongs here and was just omitted by mistake?

Reply:
I think this is incorrect. This file is the Javascript macros, not the Handlebars macros.

Reply:
So `maybeAttrs` is just a handlebars macro and it should be removed from this file completely, is that correct?

Reply:
Anyway, I removed maybeAttrs from the JS macros file, so I guess the PR is ready for the review